manager/src/main/scala/org/broadinstitute/transporter/web/WebApi.scala

```diff
@@ -206,8 +206,36 @@ class WebApi(
         )
       }
 
+  private val lookupTransfersRoute: Route[
+    (UUID, Long, Long, SortOrder),
+    ApiError,
+    Page[TransferDetails]
+  ] = singleRequestBase
+    .in("list-transfers")
+    .in(query[Long]("offset"))
+    .in(query[Long]("limit"))
+    .in(query[SortOrder]("sort"))
+    .out(jsonBody[Page[TransferDetails]])
+    .description(
+      "Get the transfer IDs for a given request ID, page number, and number of transfer IDs per page"
```

Comment:
```suggestion
      "Get transfer details for a given request"
```
the "given ..." section is redundant with the inputs spec.